github/store/db.go

```diff
@@ -47,72 +47,180 @@ const (
 )
 
 var tables = []string{
-	"organizations_versioned",
-	"users_versioned",
-	"repositories_versioned",
-	"issues_versioned",
-	"issue_comments_versioned",
-	"pull_requests_versioned",
-	"pull_request_reviews_versioned",
-	"pull_request_comments_versioned",
+	"github_organizations_versioned",
+	"github_users_versioned",
+	"github_repositories_versioned",
+	"github_issues_versioned",
+	"github_issue_comments_versioned",
+	"github_pull_requests_versioned",
+	"github_pull_request_reviews_versioned",
+	"github_pull_request_comments_versioned",
 }
 
 func (s *DB) SetActiveVersion(ctx context.Context, v int) error {
-	// TODO: for some reason the normal parameter interpolation $1 fails with
-	// pq: got 1 parameters but the statement requires 0
+	// Unified schema
 
-	_, err := s.DB.ExecContext(ctx, fmt.Sprintf(`CREATE OR REPLACE VIEW organizations AS
+	_, err := s.DB.ExecContext(ctx, "DROP MATERIALIZED VIEW IF EXISTS owners")
```

Comment:
Premature optmizzation is the root of all evil, but could it be that using https://www.postgresql.org/docs/10/sql-refreshmaterializedview.html could be benefical in terms of performances?